### PR TITLE
historyBack

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/src/webclient/actions/action_service.js [hunk 1]
```diff
@@ -540,6 +540,14 @@ function makeActionManager(env) {
                 env.services.title.setParts({ action: controller.displayName });
             }
         };
+        controller.config.onHistoryBack = () => {
```
Comment (aab-odoo) [inline addons/web/static/src/webclient/actions/action_service.js:541]: onHistoryBack -> historyBack ?

File: addons/web/static/src/legacy/action_adapters.js [hunk 2]
```diff
@@ -49,9 +49,18 @@ class ActionAdapter extends ComponentAdapter {
                 };
                 core.bus.trigger("DOM_updated");
 
+                this.__widget.el.addEventListener(
+                    "history-back",
+                    this.wowlEnv.config.onHistoryBack
+                );
+
```
Comment (aab-odoo) [inline addons/web/static/src/legacy/action_adapters.js:59]: this part is not necessary I think. Legacy widgets will use trigger_up, and owl components will directly call config.historyBack. Am I missing something?

Comment (Polymorphe57) [reply addons/web/static/src/legacy/action_adapters.js]: it is apparently but maybe only because of some tests

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**